base/Word/shl.kind

```diff
@@ -0,0 +1,3 @@
+Word.shl<size: Nat>(n: Word(size), value: Word(size)): Word(size)
+  let n_nat = Word.to_nat!(n)
+  Word.shift_right!(n_nat, value)
```

Comment:
Please don't take me wrong. Could you elaborate why `.shift_right` in the context of `shl`?

Reply:
no reason why, that's just a bug 🙂
good catch, it should be fixed now